SDL/src/Main.cpp

```diff
@@ -176,7 +175,33 @@ void processEvents()
         }
 }
 
-
+void processGPIO(int button)
+{
+    //enum Button { UP = 25, DOWN = 24, LEFT  = 23, RIGHT = 22, RELEASED = 0};
```

Comment:
We should be using the Button enum here. Also, remove this commented out code once we do that.